lib/veri_helpers.mli

```diff
@@ -0,0 +1,21 @@
+open Core_kernel.Std
```

Comment:
it looks like, that you need a simple module with only one function: 
```
module Diasm : sig
   type error = <list all possible error conditions>
   val create : arch -> t
   val insn : mem -> (t,error) Result.t
```

There is no need to provide all this small intermediate functions.